bin/anthology/anthology.py

```diff
@@ -62,6 +64,14 @@ def import_file(self, filename):
         for volume_xml in collection:
             volume = Volume.from_xml(volume_xml, collection_id, self.venues, self.sigs, self.formatter)
 
+            # skip volumes that have an ingestion date in the future
+            if datetime.strptime(volume.ingest_date, '%Y-%m-%d') > datetime.now():
+                log.info(f'Skipping volume {volume.full_id} with ingestion date {volume.ingest_date} in the future.')
+                continue
+
+            # Register the volume since we're not skipping it
+            self.venues.register(volume)
+
             if volume.full_id in self.volumes:
```

Comment:
`venues.register()` was formerly called in the `Volume` constructor (a side effect). Now, we only register volumes that don't have an ingestion date in the future.